caddyconfig/httpcaddyfile/shorthands.go

```diff
@@ -64,10 +64,13 @@ func placeholderShorthands() []string {
 		"{orig_?query}", "{http.request.orig_uri.prefixed_query}",
 		"{method}", "{http.request.method}",
 		"{uri}", "{http.request.uri}",
+		"{uri_escaped}", "{http.request.uri_escaped}",
```

Comment:
```suggestion
		"{%uri}", "{http.request.uri_escaped}",
```

And so on for the other shorthands